### ISSUE TITLE
Features used for pretrained Movie_Mcan "movie_mcan.grid.vqa2_vg"

### ISSUE THREAD

**Issue by ifmaq1** [facebookresearch/mmf#821]
## ❓ Questions and Help
Could you please tell me that, did you used **X-101** features for pretrained model of **Movie+Mcan**? and if I use a pretrained model I need the testing set for X-101 features of COCO2015 right? Testing can't be done with resnet-50 features if I use a pretrained model?

**Comment by vedanuj** [comment]
Hi @Ifrahmaqsood, Movie MCAN model uses the features available in https://github.com/facebookresearch/grid-feats-vqa#pre-trained-models-and-features, specifically the X-152++ features. It is a X-152 model. 

So if you are using a pretrained model then you will need the features available for X-152++ in the above link. Testing with any other features will not give same results.

**Comment by ifmaq1** [comment]
@vedanuj I am trying to extract the features for **test split of Coco dataset**. I used the pretrained model of **Backbone X-152++** in **https://github.com/facebookresearch/grid-feats-vqa**


to extract features I used 
**python3.6 extract_grid_feature.py --config-file configs/X-152-challenge.yaml --dataset coco_2015_test MODEL.WEIGHTS /home/grid-feats-vqa-master/output/X-152pp.pth**


**and then later on tested with pretrained model of movie_mcan**
CUDA_VISIBLE_DEVICES=0,1 mmf_predict config=projects/movie_mcan/configs/vqa2/defaults.yaml \model=movie_mcan \dataset=vqa2 \run_type=test checkpoint.resume_zoo=movie_mcan.grid.vqa2_vg \training.num_workers=2 \training.batch_size=32



**I got the following results upon submission**
[{"test-dev": {"yes/no": 83.06, "number": 34.09, "other": 56.85, "overall": 65.06}}]

that is too low then the stated results
**testdev accuracy - 73.92%**
  
  
Please let me know what am I doing wrong here

**Comment by ifmaq1** [comment]
@vedanuj any solution please?
